doc/howtos/building-images.rst

```diff
@@ -195,7 +204,7 @@ compressed formats:
 To customize the image format, modify ``local.conf``, adding the variable
 ``OSTRO_VM_IMAGE_TYPES``, set to any combination of the following::
 
-    dsk dsk.xz dsk.vdi dsk.vdi.xz
+    dsk dsk.xz dsk.zip dsk.vdi dsk.vdi.xz dsk.vdi.zip
```

Comment:
dsk.xz.bmap doesn't make sense. bmap file is generated on original, non-compressed sparse image file. Practically all compression methods are killing sparseness.

dsk.bmap can be used with multiple compressions, it applies both to dsk.xz and with development version of bmaptool also to dsk.zip.

As for dsk.vdi.bmap, it's expected that dsk.vdi not present, if dsk.vdi not explicitly in mentioned in types. Intermediate compression results are not preserved if user didn't ask for them.

And in overall, with arrival of OVA, vdi type should go away.